### DISCUSSION TITLE
Request for a Changelog

### DISCUSSION THREAD

**Discussion by batbrain9392** [giggio/node-chromedriver#423] [Ideas]
Hi! 😊

I should mention that I'm new to this package, and was recently added as a reviewer to a depandabot pr for bumping `chromedriver` in my office project.

I was wondering if it would be possible to add a changelog to this project.
`chromedriver` was recently bumped from 114.0.2 to 115.0.0, and I was wondering what changes went in, especially if there were any breaking changes and such.

Thanks for the amazing work, btw. 😊

**Comment by giggio** [comment]
Versions for this package follow Chromedriver versions. I don't remember ever having breaking versions. What could break is the support, it follows the Node.js supported versions, so when a Node.js versions is out of support I usually refactor the code to use the newer JavaScript language features, and it might break on older, unsupported versions.
What I've been doing over the years also is mostly adding functionality people request and removing packages when a native Node.js functionality supersedes it.
When I added something it was always in a no breaking way.
If something ever breaks, it should be between major versions.

Because of that, I don't think it needs a changelog... most of the time it would be: `updated Chromedriver version`.
You can see that reflected on the commits.

Regarding 114 to 115, there was a major change from the Chromedriver team, they changed URLs, it is explained in the readme, but the change should only affect people using their own mirrors of the Chromedriver's binaries.
